src/IECoreScene/MeshAlgoDistributePoints.cpp

```diff
@@ -213,6 +213,8 @@ PointsPrimitivePtr MeshAlgo::distributePoints( const MeshPrimitive *mesh, float
 
 	size_t numFaces = updatedMesh->verticesPerFace()->readable().size();
 	Generator gen( meshEvaluator.get(), uvData->readable(), faceArea, textureArea, density, densityVar, offset );
+
+	tbb::task_group_context taskGroupContext( tbb::task_group_context::isolated );
```

Comment:
This needs to be passed as a parameter in the call to `parallel_reduce()`.

Reply:
Updated in e7ac8e2